test/integration/features/cmd-config.feature

```diff
@@ -175,15 +175,7 @@ user defined options which changes default behaviour of Minishift.
     | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso        | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
     | iso-url         | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso  | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso |
     | iso-url         | centos                                                                                           | centos                                                                                          |
-       
-  @quick
-  Scenario: Setting iso-url key to a non existing path
-     When executing "minishift config set iso-url file:///home/Downloads/my_handmade_centos.iso" fails
-     Then stderr should contain
-      """
-      'file:///home/Downloads/my_handmade_centos.iso' file is not present
-      """
```

Comment:
@agajdosi If we remove this test then we wont be testing file URI for ISOs at all. So we can not remove this . However I am wondering if godog provides anyway to change a command for specific operating system.

Reply:
@LalatenduMohanty Our unit test does cover this scenario already.

Reply:
@anjannath We can not keep track of individual unit tests vs integration tests. What happens when because of some reason the unit test does not catch the issue. I remember we had once missed to catch the bug with respect to file URI and personally do not want to miss issues like that.

Reply:
@LalatenduMohanty ack, so will try to figure out how to implement this nicely to be independent of platform. Unfortunately godog does not provide anything than tags which could be used. So probably small custom step will be needed, will check that.